src/main/java/seedu/address/logic/commands/patient/EditPatientCommand.java

```diff
@@ -62,26 +61,25 @@
      * @param patientToBeEdited name in the filtered patient list to edit
      * @param editPatientDescriptor details to edit the patient with
      */
-    public EditPatientCommand(String patientToBeEdited, EditPatientDescriptor editPatientDescriptor) {
+    public EditPatientCommand(Name patientToBeEdited, EditPatientDescriptor editPatientDescriptor) {
         requireNonNull(patientToBeEdited);
```

Comment:
For multiple non-null args, we can use `requireAllNonNull(...)` instead to shorten the code!

Reply:
okay